XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +333,36 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func selectOptionFromBrowserTabMenu(view: selectMenuOption) {
```

Comment:
Hope this is better now. Are the names better for this and for the enum :)? 

Also, there are two options here becasue the BrowserTabMenu has two pages. I think there are pros and cons on both, and maybe more than the ones I say here. What do you think would be the best option @jhugman ? 
1. The one implemented, only one selectOptionFromBrowserTabMenu, with an if to check it the option selected is in the first or second tab. Pros: the code is not duplicated for BrowserTabMenu2. Cons: Need the sleep there, otherwise there is a timing issue and when checking if the element exists happens before it is there so the tests fail
2. Create a second selectOptionFromBrowserTabMenu2(view: selectMenuOption2). Pros: We would not need the sleep. Cons: more code to duplicate the selectOptionFromBrowserTabMenu and create another enum for the options in BrowserTabMenu2.

Thanks!